libvirt.go

```diff
@@ -142,6 +143,52 @@ const (
 	UndefineFlagNVRAM
 )
 
+// Capabilities returns an XML document describing the host's capabilties.
+func (l *Libvirt) Capabilities() ([]byte, error) {
```

Comment:
It seems like we should probably return a struct unmarshaled from the XML, instead of the XML itself.

Maybe return both for now with a TODO to add more fields?